src/esbmc/options.cpp

```diff
@@ -58,6 +58,9 @@ const struct group_opt_templ all_cmd_options[] = {
     {"warning,W",
      boost::program_options::value<std::vector<std::string>>(),
      ""},
+    {"dummymain,M",
```

Comment:
I don't think that `dummymain` is a good name. Specially without a good description.

You could also move to a solidity options group

Reply:
Done. 
Fixed in the latest patch.